File: solutions/beecrowd/1805/1805.cs
using System;

class Solution {
    static void Main() {
        long first, last;
        string line;
        string[] numbers;

        while ((line = Console.ReadLine()) != null) {
            numbers = line.Split(' ');
            first = Convert.ToInt64(numbers[0]);
            last = Convert.ToInt64(numbers[1]);

            Console.WriteLine((last - first + 1) * (first + last) / 2);
        }
    }
}

File: solutions/beecrowd/1805/1805.js
const readline = require('readline')

const rl = readline.createInterface({
  input: process.stdin,
  terminal: false
})

const main = () => {
  rl.on('line', (line) => {
    const [first, last] = line.split(' ').map(BigInt)
    const result = (last - first + 1n) * (first + last) / 2n

    console.log(result.toString())
  })
}

main()
